.gitignore

```
@@ -55,6 +55,8 @@ ci/
 /playwright/.auth/
 
 # Editor
+.cursor
+.vscode
```

I think it's helpful to keep `.vscode/settings.json` in source control, to promote consistent handling of things like `formatOnSave`. Perhaps we can do this:
 
```suggestion
.vscode
!.vscode/settings.json
```

---

Still, the changes made locally are going to be inside `settings.json` and I might commit them one day :(

In my experience, it's a personal decision to format on save. The most important thing the team can benefit from is is to have prettier and linting rules set up, and to make sure they are applied before opening a PR for review.